configure.com

```diff
@@ -6886,6 +6887,7 @@ $ WC "perllibs='" + perllibs + "'"
 $ WC "perlpath='" + "''vms_prefix':[000000]Perl''exe_ext'" + "'"
 $ WC "perl_static_inline='static inline'"
 $ WC "perl_symbol='" + perl_symbol + "'"  ! VMS specific
+$ WC "perl_thread_local=''" ! FIXME - as this is ia64 ABI, it may well be supported
```

Comment:
@craigberry Given this whole `__thread` thing stared with ia64, does the VMS compiler support it? If so, like accurate win32 definitions, it might be easier to update this file in a later commit.